app/jobs/schedule_authorization_request_email_job.rb

```diff
@@ -1,35 +1,43 @@
-class ScheduleAuthorizationRequestMailjetEmailJob < ApplicationJob
-  include MailjetHelpers
+class ScheduleAuthorizationRequestEmailJob < ApplicationJob
+  include MailHelpers
 
   queue_as :default
 
   attr_reader :authorization_request
 
-  def perform(authorization_request_id, authorization_request_status, mailjet_attributes)
+  def perform(authorization_request_id, authorization_request_status, mail_attributes)
     @authorization_request = AuthorizationRequest.find_by(id: authorization_request_id)
 
     return if authorization_request.blank?
     return if authorization_request_status_changed?(authorization_request_status)
 
-    deliver_mailjet_email(
-      build_message(mailjet_attributes.stringify_keys)
-    )
-  rescue Mailjet::ApiError => e
-    affect_mailjet_context_for_sentry(e, authorization_request_id)
-    raise
+    deliver_email(mail_attributes)
   end
 
   private
 
-  def build_message(mailjet_attributes)
+  def deliver_email(mail_attributes)
+    interesting_mail_attributes = {
+      to: mail_attributes[:to],
+      cc: mail_attributes[:cc]
+    }.compact
+
+    api_klass = "#{API}#{@authorization_request.api.capitalize}".constantize
+
+    mail_klass = "#{api_klass}::AuthorizationRequestMailer".constantize
+
+    mail_klass.public_send(mail_attributes[:template_name].to_sym, interesting_mail_attributes)
```

Comment:
A la lumière des différents commentaires que j'ai fait:

```suggestion
    mailer_klass.public_send(mail_attributes[:template_name], mail_recipients_attributes)
```